aravisApp/src/ADAravis.cpp

```diff
@@ -356,6 +358,7 @@ ADAravis::ADAravis(const char *portName, const char *cameraName, int enableCachi
     createParam("ARAVIS_SHIFT_BITS",     asynParamInt32,   &AravisShiftBits);
     createParam("ARAVIS_CONNECTION",     asynParamInt32,   &AravisConnection);
     createParam("ARAVIS_RESET",          asynParamInt32,   &AravisReset);
+    createParam("ARAVIS_AUTO_CONNECT", asynParamInt32, &AravisAutoConnect);
```

Comment:
Nit: not following whitespace formatting.

Reply:
Done!